checks/fifteen.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Fifteen(Test):
```

Comment:
@hljames, @brianyu28 and I discussed how we could handle pset3 (by swapping out students' `draw` function as needed). Do just discuss with him.